GVFS/GVFS.Installer.Windows/Setup.iss

```diff
@@ -616,6 +616,89 @@ begin
   Result := True;
 end;
 
+type
+  UpgradeRing = (urUnconfigured, urNone, urFast, urSlow);
+
+function GetConfiguredUpgradeRing(): UpgradeRing;
+var
+  ResultCode: integer;
+  ResultString: ansiString;
+begin
+  Result := urUnconfigured;
+  if ExecWithResult('gvfs.exe', 'config upgrade.ring', '', SW_HIDE, ewWaitUntilTerminated, ResultCode, ResultString) then begin
+    if ResultCode = 0 then begin
+      ResultString := AnsiLowercase(Trim(ResultString));
+      Log('GetConfiguredUpgradeRing: upgrade.ring is ' + ResultString);
+      if CompareText(ResultString, 'none') = 0 then begin
+        Result := urNone;
+      end else if CompareText(ResultString, 'fast') = 0 then begin
+        Result := urFast;
+      end else if CompareText(ResultString, 'slow') = 0 then begin
+        Result := urSlow;
+      end else begin
+        Log('GetConfiguredUpgradeRing: Unknown upgrade ring: ' + ResultString);
+      end;
+    end else begin
+      Log('GetConfiguredUpgradeRing: Call to gvfs config upgrade.ring failed with ' + SysErrorMessage(ResultCode));
+    end;
+  end else begin
+    Log('GetConfiguredUpgradeRing: Call to gvfs config upgrade.ring failed with ' + SysErrorMessage(ResultCode));
+  end;
+end;
+
+function IsConfigured(ConfigKey: String): Boolean;
+var
+  ResultCode: integer;
+  ResultString: ansiString;
+begin
+  Result := false

```

Comment:
I'm not sure if this matters, but it looks like elsewhere in this file we use `False` and `True` rather than `false` and `true`

Reply:
I believe it's functionally identical, but +1 for consistency.